src/core/Akka.Tests/Dispatch/ChannelExecutorSpec.cs

```diff
@@ -0,0 +1,22 @@
+﻿
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.TestKit;
+using Xunit;
+
+namespace Akka.Tests.Dispatch
+{
+    public class ChannelExecutorSpec : AkkaSpec
+    {
+        [Fact]
+        public void Should_Successfully_Create_Channel_Executor_ActorSystem()
+        {
+            var config = @"
+                          akka.actor.default-dispatcher = { 
+                              executor = channel-executor
+                              channel-executor.priority = normal
```

Comment:
Channel executor not injected when `channel-executor.priority` is not explicitly set by user @Zetanova @Aaronontheweb 
https://github.com/akkadotnet/akka.net/blob/405dfc585e633e3bf9222ed28348c325e8d1204e/src/core/Akka/Dispatch/AbstractDispatcher.cs#L123

Reply:
I made already some improvements to config loading in PR https://github.com/akkadotnet/akka.net/pull/5515

The channel-scheduler is a standalone component and `akka.channel-scheduler` is correct,
but I copied the default config block into the wrong place in the default config:
https://github.com/akkadotnet/akka.net/blob/405dfc585e633e3bf9222ed28348c325e8d1204e/src/core/Akka/Configuration/Pigeon.conf#L272-L279

Please change the default config in `Pigeon.conf` and not `akka.channel-scheduler`

Reply:
This fact needs to be documented properly, it does not reflect what we have right now in the documentation here:
https://getakka.net/articles/actors/dispatchers.html#channelexecutor